### PR TITLE
NN-3643: changing call back urls fo the prison staff hub

### PR DESCRIPTION
**Pull request by richardpopple**

### CODE REVIEW

File: backend/config.ts [hunk 1]
```diff
@@ -58,7 +58,7 @@ export const apis = {
     url: process.env.OAUTH_ENDPOINT_URL || 'http://localhost:9090/auth/',
     ui_url: process.env.OAUTH_ENDPOINT_UI_URL || process.env.OAUTH_ENDPOINT_URL || 'http://localhost:9090/auth/',
     timeoutSeconds: toNumber(process.env.API_ENDPOINT_TIMEOUT_SECONDS) || 10,
-    clientId: process.env.API_CLIENT_ID || 'elite2apiclient',
+    clientId: process.env.API_CLIENT_ID || 'prisonapiclient',
```
Comment (richardpopple) [inline backend/config.ts:61]: We now use the the prisonapiclient in local development as that that is backed by the correct callback urls in the auth app

File: backend/sessionManagementRoutes.ts [hunk 2]
```diff
@@ -94,13 +94,17 @@ export const configureRoutes = ({ app, tokenRefresher, tokenVerifier, homeLink }
...
-      res.redirect(`/login?${query}`)
+      res.redirect(`/sign-in?${query}`)
     })
   }
 
-  app.get('/login', loginMiddleware, remoteLoginIndex)
+  app.get('/sign-in', signInMiddleware, remoteSignInIndex)
 
-  app.get('/login/callback', (req, res, next) => {
+  app.get('/login/callback', (req, res) => {
```
Comment (richardpopple) [inline backend/sessionManagementRoutes.ts:103]: redirect for any book marks

File: integration-tests/integration/prisonerProfile/prisonerCellHistory.spec.js [hunk 3]
```diff
@@ -86,21 +86,15 @@ context('Prisoner cell history', () => {
         .cellDetailsLink()
         .should('contain.text', 'View details for location 1-02')
         .should('have.attr', 'href')
-        .and(
-          'include',
-          `/location-history?fromDate=2020-05-01T12:48:33&toDate=${moment().format(
-            'YYYY-MM-DDTHH:mm:ss'
-          )}&locationId=1&agencyId=MDI`
-        )
+        .and('contains', `/location-history?fromDate=2020-05-01T12:48:33&toDate=${moment().format('YYYY-MM-DDTHH:mm')}`)
```
Comment (richardpopple) [inline integration-tests/integration/prisonerProfile/prisonerCellHistory.spec.js:89]: Don't match to the second as this causes intermittent failures

File: integration-tests/integration/signIn/signIn.spec.js [hunk 4]
```diff
@@ -14,72 +14,66 @@ context('Login functionality', () => {
...
+  it('Root (/) redirects to the auth sign in page if not signed in', () => {
+    cy.task('stubSignInPage')
     cy.visit('/')
     cy.url().should('include', 'authorize')
     cy.get('h1').should('contain.text', 'Sign in')
   })
 
-  it('Login page redirects to the auth login page if not logged in', () => {
-    cy.task('stubLoginPage')
-    cy.visit('/login')
```
Comment (sp-watson) [inline integration-tests/integration/signIn/signIn.spec.js:26]: I think we should keep a `/login` test, as it is still there (albeit just a redirect)

Comment (richardpopple) [reply integration-tests/integration/signIn/signIn.spec.js:26]: yeah - your right - I'll have a look at that.

**Review by sp-watson — DISMISSED** [review]

**Review by sp-watson — APPROVED** [review]

**Pull request merged**